src/components/cc-notice/cc-notice.js

```diff
@@ -0,0 +1,262 @@
+import { css, html, LitElement } from 'lit';
+import '../cc-img/cc-img.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { dispatchCustomEvent } from '../../lib/events.js';
+import { i18n } from '../../lib/i18n.js';
+
+const closeSvg = new URL('../../assets/close-gray.svg', import.meta.url).href;
+const dangerSvg = new URL('../../assets/spam-2-fill.svg', import.meta.url);
+const infoSvg = new URL('../../assets/information-fill.svg', import.meta.url);
+const successSvg = new URL('../../assets/checkbox-circle-fill.svg', import.meta.url);
+const warningSvg = new URL('../../assets/alert-fill.svg', import.meta.url).href;
+
+/**
+ * @typedef {import('./cc-notice.types.js').NoticeIntent} NoticeIntent
+ */
+
+/**
+ * A component to display a block with a title and a message with different modes (info, success, danger, warning).
+ * You can also close it and override the message nor the icon if needed.
+ *
+ * @cssdisplay block
+ *
+ * @slot icon - Icon slot to override the default one provided.
+ * @slot message - Message slot to override the message if you want more than just a short text.
+ *
+ * @event {CustomEvent} cc-notice:dismiss - Fires an event to inform that the notice should be dismissed.
+ */
+export class CcNotice extends LitElement {
+
+  static get properties () {
+    return {
+      closeable: { type: String, reflect: true },
+      heading: { type: String, reflect: true },
+      intent: { type: String, reflect: true },
+      message: { type: String },
+      noIcon: { type: Boolean, attribute: 'no-icon', reflect: true },
```

Comment:
I don't think we need to reflect most of the attributes now that we rely on a wrapper + class system.

It's still necessary for the intent I guess unless you want to make it rely on a class system like others.